manheim_c7n_tools/version.py

```diff
@@ -17,7 +17,7 @@
 """
 
 #: The semver-compliant version of the package.
-VERSION = '0.8.6'
+VERSION = '0.8.7'
```

Comment:
My only question is the version bump - is this a patch bump or minor bump?

Reply:
Good catch, thanks. Bumped to 0.9.0